code/modules/mob/inventory/unequip.dm

```diff
@@ -0,0 +1,25 @@
+/mob/proc/u_equip(obj/item/Item)
+
+
+/mob/proc/drop_from_inventory(var/obj/item/Item, var/atom/Target = null, drop_flag = null)
+	src.u_equip(Item)
+
+	if(!Item || !Item.loc)
+		return TRUE // self destroying objects (tk, grabs)
+
+	if(!Target)
+		Target = src.loc
+		drop_flag = MOVED_DROP
```

Comment:
what is drop flag?
what possible value for it could be passed into this function?